StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -31,7 +33,8 @@
 #include "Texture/Texture.h"
 #include "Texture/TextureCubeMap.h"
 #include "Material.h"
-#include "Text.h"
+#include "TextManager.h"
+#include "QuadManager.h"
```

Comment:
I think you should include these in your GUI component.h so you don't have to include these here. If they are so core in GUI component you should in my opinion not forward declare them in h.